src/main/java/com/tictactoe/Options.java

```diff
@@ -0,0 +1,53 @@
+package com.tictactoe;
+
+public class Options {
+    private UserInput receiver;
+    private UserInputValidator validator = new UserInputValidator();
+    private String input;
+    public SelectedOptions selectedOptions = new SelectedOptions();
+
+
+    public Options (UserInput receiver) {
+        this.receiver = receiver;
+    }
+
+    public  SelectedOptions collectGamePreference(){
+        boolean allOptionsSelected = false;
+            while(!allOptionsSelected) {
+                System.out.println("Player 1 - Pick your symbol between [X] or [O]: ");
+                this.askForPlayerOneSymbol();
+                System.out.println("Player 2 - Pick your symbol between [X] or [O]: ");
+                this.askForPlayerTwoSymbol();
```

Comment:
So, can both player one and player two choose "X"? And if they do, how will the game know who wins?